command/cert.go

```diff
@@ -34,6 +37,7 @@ type CertService struct {
 	CommonName string
 	Domains    []string
 	CreateKey  bool
+	RSAKeySize string
```

Comment:
Let's accept `int` here then you can convert string to int before you initialize `CertService` struct.